honeybee_radiance/command/options/rtrace.py

```diff
@@ -0,0 +1,890 @@
+        self._u = BoolOption('u', 'uncorrelated Monte Carlo sampling - default: on')
+        self._bv = BoolOption('bv', 'back face visibility - default: on')
+        self._dt = NumericOption('dt', 'direct threshold - default: 0.030000')
+        self._dc = NumericOption('dc', 'direct certainty - default: 0.750000')
+        self._dj = NumericOption('dj', 'direct jitter - default: 0.000000',
+                                 max_value=1.0)
+        self._ds = NumericOption('ds', 'direct sampling - default: 0.200000')
+        self._dr = IntegerOption('dr', 'direct relays - default: 2')
+        self._dp = IntegerOption('dp', 'direct pretest density - default: 512')
+        self._dv = BoolOption('dv', 'direct visibility - default: on')
+        self._ss = NumericOption('ss', 'specular sampling - default: 1.000000')
+        self._st = NumericOption('st', 'specular threshold - default: 0.150000')
+        self._av = TupleOption(
+            'av', 'ambient value - default: 0.000000 0.000000 0.000000', None, 3, float
+        )
+        self._aw = IntegerOption('aw', 'ambient value weight - default: 0')
+        self._ab = IntegerOption('ab', 'ambient bounces - default: 0')
+        self._aa = NumericOption('aa', 'ambient accuracy - default: 0.100000')
+        self._ar = IntegerOption('ar', 'ambient resolution - default: 256')
+        self._ad = IntegerOption('ad', 'ambient divisions - default: 1024')
+        self._as = IntegerOption('as', 'ambient super-samples - default: 512')
+        self._me = TupleOption(
+            'me',
+            'mist extinction coefficient - default: 0.00e+000 0.00e+000 0.00e+000', None,
+            3, float)
+        self._ma = TupleOption(
+            'ma', 'mist scattering albedo - default: 0.000000 0.000000 0.000000', None,
+            3, float)
+        self._mg = NumericOption(
+            'mg', 'mist scattering eccentricity - default: 0.000000')
+        self._ms = NumericOption('ms', 'mist sampling distance - default: 0.000000')
+        self._lr = IntegerOption(
+            'lr', 'limit reflection (Russian roulette) - default: -10')
+        self._lw = NumericOption('lw', 'limit weight - default: 2.00e-003')
+        self._te = StringOption('te', 'trace excluded modifier')
+        self._tE = FileOption('tE', 'trace excluded modifiers file')
+        self._ti = StringOption('ti', 'trace included modifier')
+        self._tI = FileOption('tI', 'trace included modifiers file')
+        self._af = StringOption('af', 'ambient file')
+        self._ae = StringOption('ae', 'ambient excluded modifier')
+        self._aE = FileOption('aE', 'ambient excluded modifiers file')
+        self._ai = StringOption('ai', 'ambient included modifier')
+        self._aI = FileOption('aI', 'ambient included modifiers file')
+        self._e = FileOption('e', 'error file')
+        self._on_setattr_check = True
+
+    def on_setattr(self):
```

Comment:
It does seem a bit odd to me that the methods going by this name are not hidden.  Otherwise, if you are going to expose them, maybe they should get docstrings.

Reply:
added documentation and also made it hidden.